tools/integration_tests/util/creds_tests/creds.go

```diff
@@ -75,30 +74,39 @@ func RunTestsForKeyFileAndGoogleApplicationCredentialsEnvVarSet(testFlagSet [][]
 	}
 	operations.CloseFile(file)
 
+	return
+}
+
+func ApplyPermissionToServiceAccount(serviceAccount, permission string) {
 	// Provide permission to service account for testing.
-	_, err = operations.ExecuteGsutilCommandf(fmt.Sprintf("iam ch serviceAccount:%s:%s gs://%s", serviceAccount, permission, setup.TestBucket()))
+	_, err := operations.ExecuteGsutilCommandf(fmt.Sprintf("iam ch serviceAccount:%s:%s gs://%s", serviceAccount, permission, setup.TestBucket()))
 	if err != nil {
 		setup.LogAndExit(fmt.Sprintf("Error while setting permissions to SA: %v", err))
 	}
 	// Waiting for 2 minutes as it usually takes within 2 minutes for policy
 	// changes to propagate: https://cloud.google.com/iam/docs/access-change-propagation
 	time.Sleep(120 * time.Second)
+}
 
+func RevokePermission(gcloudArg string) {
```

Comment:
It seems this method is only executing the `gcloud` command for the given arg. Why are we renaming this method as RevokePermission?

Reply:
We haven't change function name, this method will execute command for revoking permission and throw error incase of any issue. We want this block in many places that's why I made a function out of it. original block was something like this
```
revokeFunc := func(gcloudArg string) {
		_, err := operations.ExecuteGsutilCommandf(gcloudArg)
		if err != nil {
			setup.LogAndExit(fmt.Sprintf("Error in unsetting permissions to SA: %v", err))
		}
defer revokeFunc(fmt.Sprintf("iam ch -d serviceAccount:%s:%s gs://%s", serviceAccount, permission, setup.TestBucket()))
```

Reply:
Yeah, initially it was revoking the permission. But now, it only executes the GSutil command. We need to rename this, revokePermission doesn't make sense right now.